app/controllers/hubstats/deploys_controller.rb

```diff
@@ -27,6 +27,14 @@ def index
     end
 
     def show
+      @deploy = Hubstats::Deploy.find(params[:id])
```

Comment:
You can reduce the number of trips to the database here by including the associations you know you want.

Something like `Hubstats::Deploy.find(params[:id]).includes(:repo).includes(:pull_requests)` should work.